src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandModification.java

```diff
@@ -0,0 +1,25 @@
+package org.concordion.internal.command.executeCommand.modificationStrategies;
+
+import org.concordion.api.CommandCall;
+import org.concordion.api.Element;
+import org.concordion.api.ExampleCommandCall;
+
+import java.util.List;
+
+/**
+ * Created by tim on 30/10/16.
+ */
+public abstract class ExecuteCommandModification {
+    public abstract void performModification(CommandCall commandCall, List<ExampleCommandCall> examples, List<CommandCall> beforeExamples);
+    public abstract boolean originalNodeShouldBypassExecution();
```

Comment:
Deleted. This was something I refactored out to somewhere else (into the CommandCall I believe).